plugins/paymethod/paypal/locale/cs_CZ/locale.xml

```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/paymethod/paypal/locale/cs_CZ/locale.xml
   *
-  * Copyright (c) 2014-2016 Simon Fraser University Library
-  * Copyright (c) 2006-2014 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
```

Comment:
But this is just setting the copyright dates in cs_CZ to those in every other localized version of this file.